### DISCUSSION TITLE
Arbitrary variants with breakpoints?

### DISCUSSION THREAD

**Discussion by handhikadj** [tailwindlabs/tailwindcss#11907] [Help] [answered]
I notice this doesn't work: `lg:[&_th]:text-sm` 
is this unsupported currently?

**Comment by tlgreg** [comment, marked as answer]
Should be working. [Tailwind Play](https://play.tailwindcss.com/hAuurBqicI)

**Reply by handhikadj** [reply]
nope. it doesn't work
<img width="1042" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/22522638/a57cd864-7955-49a8-993b-76199693f108">

**Reply by tlgreg** [reply]
You cut off the screenshot, is the preview area larger than the `lg` breakpoint?
